tests/unit/amo/pages/TestLandingPage.js

```diff
@@ -614,4 +627,55 @@ describe(__filename, () => {
 
     expect(root.find(HeadLinks)).toHaveLength(1);
   });
+
+  it('renders a 404 when clientApp is Android and enableFeatureStaticThemesForAndroid is false', () => {
```

Comment:
I've made a similar comment above but I'm wondering if we should specify in the description that's is for a theme only?

Reply:
Every time is calls a `renderTheme` function and we mention `enableFeatureStaticThemesForAndroid` so I don't think it is needed here.